metacat-common/src/main/java/com/netflix/metacat/common/type/TypeSignature.java

```diff
@@ -19,7 +19,7 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonValue;
-import com.google.common.base.Preconditions;
+//import com.google.common.base.Preconditions;
```

Comment:
Curious what is the issue with Preconditions on java 17?

https://guava.dev/releases/17.0/api/docs/index.html?com/google/common/base/Preconditions.html